code/modules/research/techweb/all_nodes.dm

```diff
@@ -268,8 +268,8 @@
 	id = "ai"
 	display_name = "Artificial Intelligence"
 	description = "AI unit research."
-	prereq_ids = list("robotics", "neural_programming")

```

Comment:
bad

Reply:
neural programming is already unlocked with biotech its redunant